src/api/user/userValidation.ts

```diff
@@ -13,6 +15,7 @@ export const PostUserSchema = z.object({
       username: z.string().min(5),
       organization: commonValidations.organization,
       password: commonValidations.password,
+      role: z.enum(userRolesArray),
```

Comment:
**Actionable**
@diakymenko Can we remove the role param from the request so the frontend won't be responsible for defining the role? 

The `post /users` endpoint will only be used during the initial setup, we can hardcode the role in the service. This will also prevent the frontend from breaking as it is currently not sending a role attribute with the `post /users` request.

Reply:
I have changed the role attribute to optional. In case the role is not provided the user role will be set to "Admin" by default.